### PR TITLE
Update _common.sh

### PR DESCRIPTION
**Pull request by espritlibre**
Since soupsieve has release version 2.0.0 python 3 is need.

So I try to force the soupsieve release to the last the support python2

A quick and perharps dirty hack

### CODE REVIEW

File: scripts/_common.sh [hunk 1]
```diff
@@ -20,6 +20,7 @@ install_sources() {
         cd "$final_path"
         pip install --upgrade pip
         pip install --upgrade pyramid_chameleon
+        pip install soupsieve==1.9.5
```
Comment (Josue-T) [inline scripts/_common.sh]:
```suggestion
        pip install soupsieve<2.0
```

**Comment by alexAubin** [comment]
c.f. https://github.com/mozilla-services/syncserver/commit/31d045b318c38f1f8b693ebbb518f66ea57dbfa4

**Comment by Josue-T** [comment]
Maybe it would be better to follow the master branch as there are no release sine may 2018.

**Comment by Josue-T** [comment]
Fixed by 0d2eb0626231e2f78a0c1002a4ce140c1c41fec9

**Pull request closed without merging**